osu.Game/Screens/Edit/Editor.cs

```diff
@@ -185,6 +185,7 @@ protected override IReadOnlyDependencyContainer CreateChildDependencies(IReadOnl
 
         private Bindable<float> editorBackgroundDim;
         private Bindable<bool> editorHitMarkers;
+        private Bindable<bool> editorSeekToHitobject;
```

Comment:
nit: rename to `editorSeekToHitObject` (goes everywhere else too, I'll spare us both posting the same comment everywhere)